File: usth/ICT2.2/labwork/3/Java/ButtonTest.java
class ButtonTest
{
  public static void main(String... args)
  {
    Button button = new Button("foo", "bar");
    System.out.println(button);
    button.setLabel("fu");
    button.setColor("baz");
    button.dePress();
    System.out.println(button);
  }
}

File: usth/ICT2.2/labwork/3/C++/button-test.cc
#include <iostream>

#include "button.h"

using namespace std;

int
main ()
{
  Button butt ("foo", "bar");
  butt.depress ();
  butt.undepress ();
  cout << "button " << butt.label << " color " << butt.color << endl;
}
